README.md

```diff
@@ -56,17 +56,17 @@ through a [`StakingRewards`](#stakingrewards) contract.
 
 #### `StakingRewards`
 
-`StakingRewards` is a port of [Synthetix `StakingRewards`][staking-rewards]. The changes made include:
+`StakingRewards` is a port of [Synthetix `StakingRewards`][staking-rewards]. Full diff can be found [here](https://www.diffchecker.com/9JdI2pIN/). The changes made include:
 
-- Upgrade to the Solidity version from 0.5.x to 0.8.x
+- Upgrade to the Solidity version from 0.5.x to 0.8.x.
   - It was required some reorganization of the internal structure because of changes in the inheritance resolution
     mechanism.
-  - Full diff can be found [here](https://www.diffchecker.com/rgBMl45S/).
-- Added referral code functionality for `stake()`
+- Add referral code functionality for `stake()`.
   - Referral codes are meant to be used by UIs to identify themselves as the preferred solution by users.
   - The original `stake(uint256 amount)` function still works the same as before.
   - There is a new overload `stake(uint256 amount, uint16 referral)` which performs the same operation and emits the
     new `Referral` event.
+- Update `setRewardsDuration()` to support changing the reward duration during an active distribution.
```

Comment:
Could you please add a short explanation of **why** we need such a feature, the same way we did for the referral code?

Reply:
Added

Reply:
This is more the **what** than the **why**. Maybe we should briefly mention the motivation behind it being the new splitter contract (https://github.com/makerdao/dss-flappers/pull/13) and that we want to have the possibility to set `farm.rewardsDuration = flapper.hop`.

Reply:
I'd rather avoid referring to other repos if possible and keep the more abstract explanation so the README and repo are more self-contained. But if you feel strongly about it, I can mention the Splitter (would still prefer to avoid linking to another repo though).

Reply:
Okay then, maybe just add another bullet-point with a 1-liner explanation of the issue this is aiming to solve: https://github.com/cantinasec/review-makerdao-endgametoolkit/issues/13

No need to reference the issue (it's private anyway).

Reply:
Okay, let's leave as it is.